itest/data/srv-configs/casper.internal.yaml

```diff
@@ -0,0 +1,11 @@
+cassandra:
+    seeds_file: '/code/itest/data/synapse/services/cassandra_spectre.main.json'
```

Comment:
rename the seed file to "cassandra_casper.main.json"? I don't really feel strongly through, it's pre-existing and internally we still have "cassandra_spectre" as a cluster name.